docs/kubectl-config-view.md

```diff
@@ -29,8 +29,10 @@ $ kubectl config view -o template --template='{{range .users}}{{ if eq .name "e2
 ### Options
 
 ```
+      --flatten=false: flatten the resulting kubeconfig file into self contained output (useful for making secrets)
```

Comment:
I know it's useful for making secrets, but I think its just generally useful for making self-contained kubeconfig files that can be copied across machines independent of the secret use case.